tests/utils.go

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
@aglitke @erinboyd and others: this pr ties cdi  much closer to kubevirt (same test framework, copied code, same dir structure, same make targets, etc). From day one, imo, it has been ambiguous if cdi is an independent kubernetes upstream project, or is a feature of kubevirt. We started off in our own cdi repo and now we are under kubevirt to facilitate testing. This question needs to be answered now since the answer significantly influences our reviews of this and future prs. Thanks.